src/JustEat.StatsD/Buffered/Buffer.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+
+namespace JustEat.StatsD.Buffered
+{
+    internal ref struct Buffer
+    {
+        public Buffer(Span<byte> source) : this()
```

Comment:
Last minute thought, what's with `: this()`, isn't that redundant?

Reply:
thx, good catch